### ISSUE TITLE
Multiple resizable windows make all windows resize at the same time

### ISSUE THREAD

**Issue by marbaez** [olton/Metro-UI-CSS#1205]
When i defined more than two resizable windows, always i resize the first declared window, all the windows are resizing at the same time.
See what happens here https://codepen.io/marbaez/pen/aaPvRy

Pd.:My example windows are defined in this way
```
<div data-role="window"
     data-icon="<span class='mif-rocket'></span>"
     data-title="window title"
     data-btn-close="true"
     data-btn-min="true"
     data-btn-max="true"
     data-can-resize="true"
     data-shadow="true"
     class="p-3">
  Use Font icon to create window icon.
</div>
<div data-role="window"
     data-icon="<span class='mif-rocket'></span>"
     data-title="window title"
     data-btn-close="true"
     data-btn-min="true"
     data-btn-max="true"
     data-can-resize="true"
     data-shadow="true"
     class="p-3">
  Use Font icon to create window icon.
</div>
```

Kind regards,

**Comment by olton** [comment]
Thank you. I fix it in 4.2.23. Wait for release or use develop branch

**Comment by marbaez** [comment]
Man, you're awesome and ultrafast. 
Thank you for your great job!
